src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -383,26 +386,60 @@
             <RowDefinition Height="{StaticResource HamburgerHeightGridLength}"/>
             <RowDefinition/>
         </Grid.RowDefinitions>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="2*"/>
-            <ColumnDefinition Width="1*"
-                              MinWidth="300"
-                              MaxWidth="420"/>
-        </Grid.ColumnDefinitions>
         <VisualStateManager.VisualStateGroups>
             <VisualStateGroup>
-                <VisualState x:Name="ColumnsState">
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <states:ApplicationViewModeTrigger ViewMode="DoubleLandscape"/>
+                    </VisualState.StateTriggers>
+                    <VisualState.Setters>
+                        <Setter Target="KeyGraphFeaturesControl.(Grid.Row)" Value="0"/>
+                        <Setter Target="EquationInputAreaControl.(Grid.Row)" Value="0"/>
+                        <Setter Target="GraphingNumPad.(Grid.Row)" Value="0"/>
+                        <Setter Target="KeyGraphFeaturesControl.(Grid.RowSpan)" Value="2"/>
+                        <Setter Target="EquationInputAreaControl.(Grid.RowSpan)" Value="2"/>
+                        <Setter Target="GraphingNumPad.(Grid.RowSpan)" Value="2"/>
+                        <Setter Target="KeyGraphFeaturesControl.(Grid.Column)" Value="1"/>
+                        <Setter Target="EquationInputAreaControl.(Grid.Column)" Value="1"/>
+                        <Setter Target="GraphingNumPad.(Grid.Column)" Value="0"/>
+                        <Setter Target="KeyGraphFeaturesControl.(Grid.ColumnSpan)" Value="1"/>
+                        <Setter Target="EquationInputAreaControl.(Grid.ColumnSpan)" Value="1"/>
+                        <Setter Target="GraphingNumPad.(Grid.ColumnSpan)" Value="1"/>
+                        <Setter Target="RightGrid.Margin" Value="24"/>
+                        <Setter Target="GraphingNumPad.Margin" Value="0,0,24,0"/>
+                        <Setter Target="KeyGraphFeaturesControl.Margin" Value="0,40,0,0"/>
+                        <Setter Target="EquationInputAreaControl.Margin" Value="0,40,0,0"/>
+                        <Setter Target="PaneView.TallModeConfiguration">
+                            <Setter.Value>
+                                <muxc:TwoPaneViewTallModeConfiguration>TopBottom</muxc:TwoPaneViewTallModeConfiguration>
+                            </Setter.Value>
+                        </Setter>
+                    </VisualState.Setters>
+                </VisualState>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <states:ApplicationViewModeTrigger ViewMode="DoublePortrait"/>
```

Comment:
Weird layout when rotating in double portrait
![image](https://user-images.githubusercontent.com/6334170/76481508-ea16d880-63ce-11ea-82f5-7cba5571e758.png)

Reply:
same issue than above, probably the state trigger not in sync with the twopaneview